modules/nw-networkpolicy-audit-disable.adoc

```diff
@@ -0,0 +1,45 @@
+[id="nw-networkpolicy-audit-disable_{context}"]
+= Disabling network policy audit logging
+
+As a cluster administrator, you can disable network policy audit logging for a namespace.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in to the cluster with a user with `cluster-admin` privileges.
+
+.Procedure
+
+. To disable network policy audit logging for a namespace, enter the following command:
++
+[source,terminal]
+----
+$ annotate --overwrite namespace <namespace> k8s.ovn.org/acl-logging={}
+----
++
+--
+where:
+
+`<namespace>`:: Specifies the name of the namespace.
+--
++
+[TIP]
+====
+You can alternatively apply the following YAML to disable audit logging:
```

Comment:
Despite the fact that I don't decide, I'll hazard my opinion that there's any value to showing YAML for removing resources.  Any customer practicing GitOps just removes the annotation from the original and redeploys.

Reply:
@mikemckiernan, I actually shameless stole this idea from here: https://github.com/openshift/openshift-docs/pull/32817/files#diff-8c60687ef0e3ab77cb88c06c0ba409f89a0eb80e6719e462615be3c3a3665e9dR40